app/main.py

```diff
@@ -1,2 +1,81 @@
+import dataclasses
+from typing import Any
+
+
+@dataclasses.dataclass()
+class Node:
+    key: Any
+    value: Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.size = 0
+        self.threshold = round(self.capacity * 2 / 3)
+        self.hash_table = [None] * self.capacity
+
+    def collision(self, node: Node, node_ind: int) -> int:
+        curr_id = node_ind
```

Comment:
Do not abbreviate words if it is not really necessary, it saves you only few symbols. It will make code more readable